index.html

```diff
@@ -20,7 +20,7 @@
     <a class="item portfolio" href="#portfolio">portfolio</a>
     <a class="item services" href="#services">services</a>
     <a class="item team" href="#team">team</a>
-    <a class="right item">contact</div>
+    <a class="right item" href="mailto:Cortlandtlabs@gmail.com">contact</div>
```

Comment:
This is the contact you referred to in the issue, or were you talking about something else